base/deprecated.jl

```diff
@@ -80,7 +80,7 @@ function firstcaller(bt::Array{Ptr{Void},1}, funcsym::Symbol)
         if lkup === ()
             continue
         end
-        fname, file, line, fromC = lkup
+        fname, file, line, inlined_file, inlined_line, fromC = lkup
```

Comment:
You are right, the variable names were not good. I renamed them.

- `file` and `line` always reflect where code at a given address was defined, inlined or otherwise.
- `inlinedat_file` and `inlinedat_line` (renamed) reflect where that code ended up.

With updates, here is an example backtrace:
```
julia> foo()
ERROR: foobar
 [inlined code] from show.jl:110
 in foo at /cmn/jlbugs/1334_macro_lnos/foo.jl:5
```

for the following function:
```
function foo()
    @show error("foobar")
end
```